java/src/main/java/com/saucelabs/saucebindings/SauceSession.java

```diff
@@ -2,45 +2,44 @@
 
 import lombok.Getter;
 import lombok.Setter;
-import org.openqa.selenium.InvalidArgumentException;
+import org.openqa.selenium.JavascriptExecutor;
 import org.openqa.selenium.MutableCapabilities;
 import org.openqa.selenium.remote.RemoteWebDriver;
 
-import java.net.MalformedURLException;
 import java.net.URL;
 
 public class SauceSession {
     @Getter @Setter private DataCenter dataCenter = DataCenter.US_WEST;
     @Getter private final SauceOptions sauceOptions;
-    @Setter private URL sauceUrl;
 
     @Getter private RemoteWebDriver driver;
+    @Getter APIStrategy sauceAPIStrategy;
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/SauceLabs/sauce_bindings/pullRequest?prid=6386972)